lib/typescript/assets/images/icons/arrowDown.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="9" viewBox="0 0 14 9" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
if we get rid of the width and height and just keep the viewBox, we have the possibility to resize via css without cutting of edges.